skins/oasis/css/core/ads.scss

```diff
@@ -180,6 +180,14 @@ $uap-video-animation-duration: 0.4s;
 .video-player {
 	position: relative;
 
+	&.autoplay {
+		z-index: 1000;
```

Comment:
Video is playing above image not instead of image on the latest version of code.

why do we have this here? I removed it and video worked just fine.